### PR TITLE
Don't try to save something for a button which is neither a checkbox nor a radio

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: test/unit/annotation_spec.js [hunk 1]
```diff
@@ -2317,6 +2317,28 @@ describe("annotation", function () {
...
+        pdfManagerMock,
+        idFactoryMock
+      )
+        .then(annotation => {
+          return annotation.save(partialEvaluator, task, {});
+        }, done.fail)
+        .then(data => {
+          expect(data).toEqual(null);
+          done();
+        }, done.fail);
```
Comment (Snuffleupagus) [inline test/unit/annotation_spec.js:2340]: Nit: Do you actually need to repeat the `done.fail` part multiple times here?
Unless I'm missing something obvious here, it ought to be possible to simply add one `catch` at the end to deal with failures:
```suggestion
        .then(annotation => {
          return annotation.save(partialEvaluator, task, {});
        })
        .then(data => {
          expect(data).toEqual(null);
          done();
        })
        .catch(done.fail);
```

**Comment by timvandermeij** [comment]
Looks good; thanks!

**Pull request merged**